analyzers/tests/SonarAnalyzer.UnitTest/TestCases/CollectionsShouldImplementGenericInterface.CSharp10.cs

```diff
@@ -5,9 +5,9 @@
 
 namespace Tests.Diagnostics
 {
-    record struct TestRecordStruct_01 : IList { } // FN
+    record struct TestRecordStruct_01 : IList { } // Noncompliant {{Refactor this collection to implement 'System.Collections.Generic.IList<T>'.}}
+    //            ^^^^^^^^^^^^^^^^^^^
```

Comment:
You know that the message assertion and the `^^^^` markers are optional because they are already in the main test file, right?
It makes no harm having it, it's just to be clear that it is not required.